src/Http/GraphCollectionRequest.php

```diff
@@ -78,9 +79,10 @@ class GraphCollectionRequest extends GraphRequest
     * @param string $apiVersion   The version of the API to call
     * @param string $proxyPort    The url where to proxy through
     * @param bool $proxyVerifySSL Whether the proxy requests should perform SSL verification
+     * @param Client|null $guzzleClient The url where to proxy through
```

Comment:
the description is incorrect

Reply:
Sorry about that. I can work on a PR to fix the description & allow client injection. Is that ok?

Reply:
Np. Maybe hold off for now, I am going to give the v2 RC a try 😉